articles/api-management/api-management-policies.md

```diff
@@ -17,6 +17,10 @@ More information about policies:
 + [Set or edit policies](set-edit-policies.md)
 + [Policy expressions](api-management-policy-expressions.md)
 
+> [!IMPORTANT]
+>  [Limit call rate by subscription](api-management-access-restriction-policies.md#LimitCallRate) and [Set usage quota by subscription](api-management-access-restriction-policies.md#SetUsageQuota) have a dependency on the subscription key. All other policies can be utiized with bearer token or subscription key. 
+
```

Comment:
@HalaaM - Thanks for submitting. The second sentence makes it sound like bearer token and subscription key are the only cases to consider, but there are others like certs, etc. Would it be more general to say something like " A subscription key isn't required when other policies are used."?